src/nginxconfig/templates/setup_sections/ssl_cert.vue

```diff
@@ -159,13 +183,54 @@ THE SOFTWARE.
                         ].filter(x => x !== null).join(' ')
                     )).join('\n');
             },
+            acmeShCmds() {
+                return this.$props.data.domains
+                    .filter(domain => domain.https.certType.computed === 'letsEncrypt')
+                    .map(domain => (
+                        [
+                            'acme.sh --issue',
+                            `-d ${domain.server.domain.computed}`,
+                            domain.server.wwwSubdomain.computed ? `-d www.${domain.server.domain.computed}` : null,
+                            domain.server.cdnSubdomain.computed ? `-d cdn.${domain.server.domain.computed}` : null,
+                            `-w ${domain.server.path.computed}${domain.server.documentRoot.computed}`,
```

Comment:
Is there a reason we're writing to the domain's public assets directory,  rather than the shared lets encrypt webroot directory that the equivalent certbot command uses, and is designed for this purpose in the tool? (`this.letsEncryptDir`)